src/DataFrames.jl

```diff
@@ -16,6 +16,8 @@ using Reexport
 using GZip
 using SortingAlgorithms
 
+using FileIO  # remove after read_rda deprecation period
```

Comment:
FileIO recognizes the R data format by itself, so `FileIO.load` should work without RData. As for why the dependency is FileIO instead of RData, I had initially set it to RData but changed it to FileIO and I forget why. 😕